terraform/main.tf

```diff
@@ -68,3 +75,115 @@ module "development" {
   task_definition_secrets      = {}
   task_definition_secret_count = 0
 }
+
+module "staging" {
+  # We pin to `master` for now, until we have tagged releases of the modules.
+  source = "github.com/LBHackney-IT/aws-mat-components-per-service-terraform.git//modules/environment/frontend?ref=master"
+
+  environment_name = "staging"
+  application_name = "thc"
+
+  security_group_name_prefix = "mat-frontend-sg"
+  lb_prefix                  = "hackney-ext-mat-alb"
+
+  # Note that this process should only use ports in the 200X range to avoid port
+  # clashes.
+  port = 2002
+
+  listener_port = 80
+  path_pattern  = "/thc*"
+
+  desired_number_of_ec2_nodes = 2
+
+  ecs_execution_role = "${data.aws_iam_role.ecs_task_execution_role.arn}"
+  lb_iam_role_arn    = "${data.aws_iam_role.ec2_container_service_role.arn}"
+
+  task_definition_environment_variables = {
+    NODE_ENV = "production"
+
+    ENVIRONMENT_NAME = "staging"
+
+    PROCESS_NAME       = "thc"
+    BASE_PATH          = "/thc"
+    PROCESS_TYPE_VALUE = "100000156"
+    PROCESS_TYPE_NAME  = "Tenancy and household check"
+
+    PROCESS_API_HOST     = "dg5hz8mgwc.execute-api.eu-west-2.amazonaws.com"
+    PROCESS_API_BASE_URL = "/staging/mat-process/api"
+
+    MAT_API_HOST     = "g6bw0g0ojk.execute-api.eu-west-2.amazonaws.com"
+    MAT_API_BASE_URL = "/staging/manage-a-tenancy-api"
+  }
+
+  task_definition_environment_variable_count = 10
+
+  task_definition_secrets = {
+    PROCESS_API_JWT_SECRET  = "${local.parameter_store}/staging-thc-ROCESS_API_JWT_SECRET"
+    PROCESS_API_KEY         = "${local.parameter_store}/staging-thc-PROCESS_API_KEY"
+    MAT_API_JWT_SECRET      = "${local.parameter_store}/staging-thc-MAT_API_JWT_SECRET"
+    MAT_API_DATA_SHARED_KEY = "${local.parameter_store}/staging-thc-MAT_API_DATA_SHARED_KEY"
+    MAT_API_DATA_SALT       = "${local.parameter_store}/staging-thc-MAT_API_DATA_SALT"
+    MAT_API_DATA_ITERATIONS = "${local.parameter_store}/staging-thc-MAT_API_DATA_ITERATIONS"
+    MAT_API_DATA_KEY_SIZE   = "${local.parameter_store}/staging-thc-MAT_API_DATA_KEY_SIZE"
+    MAT_API_DATA_ALGORITHM  = "${local.parameter_store}/staging-thc-MAT_API_DATA_ALGORITHM"
+    MAT_API_DATA_IV         = "${local.parameter_store}/staging-thc-MAT_API_DATA_IV"
+  }
+
+  task_definition_secret_count = 9
```

Comment:
Can we add these to the `development` environment, too, please?

Reply:
Yes sure, we'll add these later. We didn't want to touch development environment at this stage because updating the values will trigger destroy and create cycle for that resource which need to be managed.

Reply:
Ok, that makes sense, I guess.